modules/channel-teams/src/backend/client.ts

```diff
@@ -31,29 +31,29 @@ export class TeamsClient {
   async initialize() {
     if (!this.config.appId || !this.config.appPassword) {
       return this.logger.error(
-        'You need to set the appId and the appPassword before enabling this channel. It will be disabled for this bot.'
+        `[${this.botId}] You need to set the appId and the appPassword before enabling this channel. It will be disabled for this bot.`
```

Comment:
`this.logger.forBot(this.botId).error()`, this way the log will also be displayed in the logs on the studio, and it will be displayed like all bot-related logs

Reply:
This is already the case, isn't it?

```ts
constructor(private bp: typeof sdk, private botId: string, private config: Config, private publicPath: string) {
    this.logger = bp.logger.forBot(this.botId)
}
```